pkg/apis/pingcap.com/v1alpha1/types.go

```diff
@@ -137,12 +137,13 @@ type TiDBSlowLogTailerSpec struct {
 // TiKVSpec contains details of PD member
 type TiKVSpec struct {
 	ContainerSpec
-	Replicas         int32               `json:"replicas"`
-	Affinity         *corev1.Affinity    `json:"affinity,omitempty"`
-	NodeSelector     map[string]string   `json:"nodeSelector,omitempty"`
-	StorageClassName string              `json:"storageClassName,omitempty"`
-	Tolerations      []corev1.Toleration `json:"tolerations,omitempty"`
-	Annotations      map[string]string   `json:"annotations,omitempty"`
+	Privileged           bool                `json:"privileged,omitempty"`
```

Comment:
what about add an annotions to `TidbCluster` instead of a `Privileged` attribute?

```
    pingcap.com/tikv-privileged: "true"
```